aws/sqs/variables.tf

```diff
@@ -0,0 +1,38 @@
+variable "stack" {
+  description = "The name of the stack"
+  type        = string
+}
+
+variable "env" {
+  description = "The name of the environment"
+  type        = string
+}
+
+variable "queue_name" {
+  description = "The shorthand name of the queue. The full queue name can be retrieved as an output. Note that an empty string is still a valid queue name."
+  type        = string
+}
+
+variable "visibility_timeout_seconds" {
+  description = "The amount of time allowed to the processor to process a message before it is declared failed. Defaults to 30 seconds."
+  type        = number
+  default     = 30
+}
+
+variable "receive_wait_time_seconds" {
+  description = "The time to wait when polling for new messages. Use 0 for immediate response. Longer values are preferred. AWS recommends a maximum of 20 seconds."
+  type        = number
+  default     = 5
```

Comment:
why shorten this from 20?

Reply:
I shortened it because 20 is the maximum, and I think the configuration we were looking at in perfected was 10 seconds. Thinking about this a bit more, and doing a bit more reading, I think it makes sense to set it to a large value, and have those that need to adjust downward.